src/ssh.c

```diff
@@ -187,14 +193,26 @@ int dcc_ssh_connect(char *ssh_cmd,
                     pid_t *ssh_pid)
 {
     pid_t ret;
-    char *child_argv[10];
-    int i;
+    const int max_ssh_args = 12;
+    char *ssh_args[max_ssh_args];
+    char *child_argv[10+max_ssh_args];
+    int i,j;
```

Comment:
Having got to the end of the function.  Is the `j` itterater needed?  It looks like `i` could be used in its place for the current code.